compiler-rt/test/builtins/Unit/extendhfxf2_test.c

```diff
@@ -5,59 +5,62 @@
 #include <math.h> // for isnan, isinf
 #include <stdio.h>
 
-#include "int_lib.h"
+#include "fp_test.h"
 
-#if HAS_80_BIT_LONG_DOUBLE && defined(COMPILER_RT_HAS_FLOAT16)
+#if __LDBL_MANT_DIG__ == 64 && defined(__x86_64__) &&                          \
```

Comment:
Keep the old version and change the one in fp_test.h instead.

Reply:
Ok. Sure.

Reply:
Changing back to check `#if __LDBL_MANT_DIG__ == 64 && defined(__x86_64__)` as fp80 functions are exclusive for x86_64 and __uint128t being unavailable in some configurations.